### PR TITLE
Update sample app

### PR DESCRIPTION
**Pull request by mehranrashid**
- Add default values
- Remove redundant code
- In-line with version 1 release

Proof of training run:
 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Install additional requirements:
 ```bash
 pip install torch
 pip install protobuf==3.20
+pip install pandas
```
Comment (robg-answer) [inline README.md:33]: Do we need boto3

Comment (mehranrashid) [reply README.md:33]: No there's no reference to using boto3 in the sample app

**Review by robg-answer — APPROVED** [review]

**Review by yorkeanswer — APPROVED** [review]
Nice, could this branch also get merged into `main` which is tracking `v1`?

**Comment by mehranrashid** [comment]
> Nice, could this branch also get merged into `main` which is tracking `v1`?

My idea was to merge it into develop, and then merge develop into main, that way both branches will have the changes?

**Pull request merged**